lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -453,6 +464,28 @@ def cmd_search(*args)
             true
           end
 
+          def colorize_module_rank(rank)
+            case rank
+            when ManualRanking
+              # Extra spaces are needed because dark colors shift the rest of the row
```

Comment:
Colours? ;)

Reply:
Haha, obligatory [internationalization sucks](https://youtu.be/0j74jcxSunY)